app/mailers/activity_mailer.rb

```diff
@@ -2,17 +2,20 @@
 #
 # @api private
 class ActivityMailer < ApplicationMailer
+  helper :application_notifications
+
   # Emails a recipient, informing him of an activity.
   #
   # @param [User] recipient The recipient of the email.
   # @param object The object to be made available to the view, accessible using +@object+.
-  # @param [String] view_path The path to the view which should be rendered.
-  def email(recipient, object, view_path)
+  # @param [Course::Notification|UserNotification] notification The notification which needs to
+  #   send out an email
+  def email(recipient, object, notification)
     @recipient = recipient
     @object = object
     mail(to: recipient.email) do |format|
-      format.html { render view_path }
-      format.text { render view_path }
+      format.html { render view_context.notification_view_path(notification) }
```

Comment:
move `view_context.notification_view_path` to a private method, better code readability + can stub it in tests